### PR TITLE
fix: add new busy indicators

### PR DESCRIPTION
**Pull request by saad-mo**
- added new `fd-loading-spinner`
- added new `fd-loading-dots`
- marked `fd-spinner` as deprecated 
- updated documentation 

#### Test

* http://localhost:3030/loading-spinner
* http://localhost:3030/loading-dots
* http://127.0.0.1:4000/components/loading-spinner.html

New Loading Spinner
Screen Shot 2019-06-25 at 2 28 06 PM

New Loading Dots
Screen Shot 2019-06-25 at 2 28 16 PM

#### Changelog

**New**

* `fd-loading-spinner`
* `fd-loading-dots`

**Changed**

* `fd-spinner` (deprecated)

**Removed**

nothing

### CODE REVIEW

**Comment by droshev** [comment]
@saad-mo can you bring the description from fundmantal-ui, resolve the conflicts and try to make the component self-contained?

File: docs/pages/components/loading-spinner.md [hunk 1]
```diff
@@ -30,24 +41,17 @@ The loading element is used to display the loading indicator animation.
...
-    <div class="fd-panel__header">
-        <h3 class="fd-panel__title">Lorem ipsum</h3>
-    </div>
-    <!-- Loaded content goes here -->
-    <div class="fd-panel__footer">
-        <p><em>Etiam pulvinar turpis sed velit porttitor vel adipiscing velit fringilla.</em></p>
-    </div>
+
+
+## Spinner (deprecated)
```
Comment (jbadan) [inline docs/pages/components/loading-spinner.md]: We should just remove this. We're about to release a breaking change so might as well.

File: scss/components.scss [hunk 2]
```diff
@@ -44,6 +44,9 @@
 @import "components/popover";
 @import "components/time";
 @import "components/localization-editor";
+@import "components/loading-dots";
+@import "components/loading-spinner";
+@import "components/combobox-input";
```
Comment (jbadan) [inline scss/components.scss]: combobox-input shouldn't be here

File: test/templates/index.njk [hunk 3]
```diff
@@ -108,6 +108,15 @@ li {
           <li><a href="pages/grid">Grid</a></li>
       </ul>
 
+      <h2>Deprecated</h2>
```
Comment (jbadan) [inline test/templates/index.njk]: just delete - let's not maintain unnecessary code

File: test/templates/loading-spinner/index.njk [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+{% from "../button/component.njk" import button %}
```
Comment (jbadan) [inline test/templates/loading-spinner/index.njk]: Where is button being used? We shouldn't be mixing components in the playground.

**Comment by jbadan** [comment]
Work needs to be migrated to 0.3.0 specs.

**Pull request closed without merging**